docs/layouts/shortcodes/generated/optimizer_config_configuration.html

```diff
@@ -89,5 +89,17 @@
             <td>Boolean</td>
             <td>When it is true, the optimizer will collect and use the statistics from source connectors if the source extends from SupportsStatisticReport and the statistics from catalog is UNKNOWN.Default value is true.</td>
         </tr>
+        <tr>
+            <td><h5>table.optimizer.busy-join-reorder</h5><br> <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span></td>
```

Comment:
This option can be removed, since we can use `table.optimizer.busy-join-reorder-dp-threshold` <= 0 to disable busy-join-reorder

Reply:
> This option can be removed, since we can use `table.optimizer.busy-join-reorder-dp-threshold` <= 0 to disable busy-join-reorder

Done!